kola/tests/util/devcontainer.go

```diff
@@ -64,6 +64,11 @@ function download_dev_container_image {
         version=$(source /usr/share/flatcar/release; echo "${FLATCAR_RELEASE_VERSION}")
         image_url=$(process_template '{{ .ImageDirectoryURLTemplate }}/flatcar_developer_container.bin.bz2' "${arch}" "${version}")
 
+        if [ "$(curl -I --retry-delay 1 --retry 60 --retry-connrefused --retry-max-time 60 --connect-timeout 20 -L -s -o /dev/null -w "%{http_code}" "${image_url}")" = 404 ]; then
```

Comment:
Yes, we don't skip it because I didn't want to duplicate the URL assembly but if wanted, we can do this. `SkipFunc` is not enough because we need the version ID from the image's os-release file. I can rework this to skip the test.